manage_proj_cat_delete.php

```diff
@@ -68,7 +68,7 @@
 # Protect the 'default category for moves' from deletion
 $t_default_category_id = config_get( 'default_category_for_moves', /* default */ null, ALL_USERS, ALL_PROJECTS );
 $t_query = 'SELECT count(config_id) FROM {config} WHERE config_id = ' . db_param() . ' AND value = ' . db_param();
-$t_default_cat_count = db_result( db_query( $t_query, array( 'default_category_for_moves', $f_category_id ) ) );
+$t_default_cat_count = db_result( db_query( $t_query, array( 'default_category_for_moves', (string)$f_category_id ) ) );
```

Comment:
Ah I missed the type of config check.  At least, it feels that this logic should moved to category API.  That can in turn leverage a new API in config api.